test/all.bash

```diff
@@ -111,7 +116,7 @@ for dir in unit lib; do
     td=$VIRGIL_LOC/test/$dir
     print_line
     echo "${CYAN}($V3C_STABLE) $dir${NORM}"
-    (cd $td && AENEAS_TEST=$V3C_STABLE $td/test.bash)
+    (cd $td && AENEAS_TEST=$V3C_STABLE PROGRESS_ARGS=$PROGRESS_ARGS $td/test.bash)
```

Comment:
I believe `export PROGRESS_ARGS` will make the explicit passing unnecessary.

Reply:
Yep. Agreed.

Reply:
This is not required any more if the `PROGRESS_ARGS` variable is set directly like in this comment: https://github.com/titzer/virgil/pull/110#discussion_r1038317860